general_rules/Organization.tex

```diff
@@ -110,34 +110,33 @@ \subsection{Schedule}
 	\end{tabular}
 
 	\caption{Example schedule.
-		Each team has assigned at least two test slots in every block.
-		At least two blocks are scheduled per day with an assigned theme.
-		A team can choose a different task in each test, meaning at least 4 different tests per stage.
+		Each team has two test slots in every block.
+		Two blocks are scheduled per day with an assigned theme.
+		A team can choose a different task in each test, meaning at most 4 different tests per stage.
```

Comment:
Is this correct ("at most 4")? I would think that, given the repetition of tests in each block, a team could carry out a different test in each repetition if they wanted to.

Reply:
No, this is wrong.

2 blocks per scenario (2) with **at least** 2 attempts each means **at least** 4 tests per day, and at least 8 tests in Stage I.
Stage II might be less, as usual

Reply:
Well I took it as this description specifically referring to the example given in the graphic which would mean at most 4 tests. The at least thing is I think already mentioned in the more general description before a specific example is made

Reply:
Seems good to me now.